src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -26,115 +26,115 @@ static void beforeAll() {
 
     @BeforeEach
     void setUp() {
-        mark = new User("Mark","123456", 18);
+        testUser = new User("Mark","123456", 18);
     }
 
     @Test
-    void register_setNullUser_NotOk() {
+    void register_SetNullUser_NotOk() {
```

Comment:
the first letter after _ in lowerCase (check it everywhere) 
<img width="635" alt="image" src="https://user-images.githubusercontent.com/96740815/198008021-2953724e-96a1-4907-938c-aec0435f181f.png">